scripts/drupal-vm/config.yml

```diff
@@ -88,7 +88,7 @@ php_packages_extra:
   - imagemagick
 
 # XDebug configuration.
-php_xdebug_version: 2.5.0
+php_xdebug_version: 2.6.0
```

Comment:
why not `2.6.1`?

Reply:
no strong opinion, i updated to 2.6.0 based on some older DrupalVm docs, happy to increment to 2.6.1